aggregator/src/app/AggregationStrategy.ts

```diff
@@ -235,11 +235,16 @@ export default class AggregationStrategy {
       let success: boolean;
 
       if (bundleResult.success) {
-        const [operationResults] = bundleResult.returnValue;
+        const [operationStatuses, results] = bundleResult.returnValue;
+
+        // TODO: is it possible to decode this?
+        // Here is an example when I try to send more ETH than an account has:results =
+        // results = [ "0x4e487b710000000000000000000000000000000000000000000000000000000000000011" ]
```

Comment:
Would the work in https://github.com/web3well/bls-wallet/pull/366 help with this?